scripts/lib/message.go

```diff
@@ -87,7 +88,8 @@ var reMsgFilename = regexp.MustCompile(`^(\d{4})-(\d{2})-\d{2}\.json$`)
 
 // ReadLogFile : pathに指定したJSON形式のメッセージデータを読み込む。
 // すでにそのファイルが読み込み済みの場合は処理をスキップする。
-func (m *MessageTable) ReadLogFile(path string) error {
+// readAllMessagesがfalseである場合は特定のサブタイプを持つメッセージのみをmsgMapに登録する。
```

Comment:
```suggestion
// デフォルトでは特定のサブタイプを持つメッセージのみをmsgMapに登録するが、
// readAllMessages が true である場合はすべてのメッセージを登録する。
```

一般的な(ゼロコンフィグ時の)動作を説明してから、
特殊な動作を説明してくれたほうが読む人にはわかりやすいので。

Reply:
#44 で修正